Formula/fastani.rb

```diff
@@ -20,6 +21,7 @@ class Fastani < Formula
   uses_from_macos "zlib"
 
   on_macos do
+    depends_on "gcc@9"
```

Comment:
```suggestion
```
`fastani` uses `libomp` for OpenMP support in Clang, rather than using GCC, so it does not depend on GCC.
See PR https://github.com/brewsci/homebrew-bio/pull/1088/files
Because of that, this PR is no longer needed.